software/gobookmarks.yml

```
@@ -0,0 +1,11 @@
+name: gobookmarks
+website_url: https://github.com/arran4/gobookmarks
+source_code_url: https://github.com/arran4/gobookmarks
+description: Landing page to display bookmarks stored in GitHub, GitLab or local Git.
+licenses:
+  - AGPL-3.0
+platforms:
+  - Go
+  - Docker
+tags:
+  - Bookmarks and Link Sharing
```

Seems to be a better fit.
```suggestion
  - Personal Dashboards
```

---

Definitely not link sharing. But it was literally born from exported bookmarks. Happy to accept either though. I will accept now push back if you think my reasoning implies the other is better.